docs/pipeline-components-and-applications/loaders-storage-targets/snowflake-streaming-loader/configuration-reference/_common_config.md

```diff
@@ -0,0 +1,97 @@
+</tr>
+<tr>
+    <td><code>skipSchemas</code></td>
+    <td>Optional, e.g. <code>["iglu:com.example/skipped1/jsonschema/1-0-0"]</code> or with wildcards <code>["iglu:com.example/skipped2/jsonschema/1-*-*"]</code>. A list of schemas that won't be loaded to Snowflake. This feature could be helpful when recovering from edge-case schemas which for some reason cannot be loaded to the table.</td>
+</tr>
+<tr>
+    <td><code>monitoring.metrics.statsd.hostname</code></td>
+    <td>Optional. If set, the loader sends statsd metrics over UDP to a server on this host name.</td>
+</tr>
+<tr>
+    <td><code>monitoring.metrics.statsd.port</code></td>
+    <td>Optional. Default value 8125. If the statsd server is configured, this UDP port is used for sending  metrics.</td>
+</tr>
+<tr>
+    <td><code>monitoring.metrics.statsd.tags.*</code></td>
+    <td>Optional. A map of key/value pairs to be sent along with the statsd metric.</td>
+</tr>
+<tr>
+    <td><code>monitoring.metrics.statsd.period</code></td>
+    <td>Optional. Default <code>1 minute</code>. How often to report metrics to statsd.</td>
+</tr>
+<tr>
+    <td><code>monitoring.metrics.statsd.prefix</code></td>
+    <td>Optional. Default <code>snowplow.snowflake-loader</code>. Prefix used for the metric name when sending to statsd.</td>
+</tr>
+<tr>
+    <td><code>monitoring.webhook.endpoint</code></td>
+    <td>Optional, e.g. <code>https://webhook.example.com</code>.  The loader will send to the webhook a payload containing details of any error related to how Snowflake is set up for this loader.</td>
+</tr>
+<tr>
+    <td><code>monitoring.webhook.tags.*</code></td>
+    <td>Optional. A map of key/value strings to be included in the payload content sent to the webhook.</td>
+</tr>
+<tr>
+    <td><code>sentry.dsn</code></td>
+    <td>Optional. Set to a Sentry URI to report unexpected runtime exceptions.</td>
+</tr>
+<tr>
+    <td><code>sentry.tags.*</code></td>
+    <td>Optional. A map of key/value strings which are passed as tags when reporting exceptions to Sentry.</td>
+</tr>
+<tr>
+    <td><code>telemetry.disable</code></td>
+    <td>Optional. Set to <code>true</code> to disable <Link to="/docs/getting-started-on-community-edition/telemetry/">telemetry</Link>.</td>
+</tr>
+<tr>
+    <td><code>telemetry.userProvidedId</code></td>
+    <td>Optional. See <Link to="/docs/getting-started-on-community-edition/telemetry/#how-can-i-help">here</Link> for more information.</td>
+</tr>
+<tr>
+    <td><code>output.good.jdbcLoginTimeout</code></td>
+    <td>Optional. Sets the login timeout on the JDBC driver which connects to Snowflake</td>
+</tr>
+<tr>
+    <td><code>output.good.jdbcNetworkTimeout</code></td>
+    <td>Optional. Sets the network timeout on the JDBC driver which connects to Snowflake</td>
+</tr>
+<tr>
+    <td><code>output.good.jdbcQueryTimeout</code></td>
+    <td>Optional. Sets the query timeout on the JDBC driver which connects to Snowflake</td>
+</tr>
```

Comment:
Just pointing out these `output.good.jdbc*Timeout`s should really belong in the Snowflake configuration section.  But they are of such low importance that I decided to move them to a less valuable part of the page.